src/MusicStore/Components/AnnouncementComponent.cs

```diff
@@ -25,6 +26,13 @@ public IMemoryCache Cache
             set;
         }
 
+        [Activate]
+        public ISystemClock SystemClock
```

Comment:
No, just redefine it. Don't use it from any other assembly because these interfaces are not contractual.